tests/tasks.py

```diff
@@ -3,18 +3,36 @@
 import os
 import os.path
 from time import sleep
-
 from invoke import task
 
 
-GLIDE_ARCH = 'linux-amd64'
-
-
 @task
-def glide(c, arch=GLIDE_ARCH, version='v0.13.2'):
-    """Dowload glide."""
+def glide(c, version='v0.13.2'):
+    """Download glide."""
     print('Downloading glide')
 
+    # find glide version, copied from glide installation script
+    os_name = c.run('uname|tr "[:upper:]" "[:lower:]"', hide='stdout')
+    os_name = os_name.stdout.replace('\n', '')
+    arch_cmd = """
+        ARCH=$(uname -m)
```

Comment:
this is way overkill. we won't run this other than on linux and osx. also it comes out much more complex in bash than transalting the logic to pure python:
```python
if os_name == 'linux':
    arch = 'linux-amd64'
elif os_name == 'darwin':  # or whatever the result is on osx:
    arch = 'darwin-amd64'  # or whatever should be here
else:
    raise invoke.exceptions.Faliure(...)
```
see http://docs.pyinvoke.org/en/latest/api/exceptions.html#invoke.exceptions.Failure